src/application/services/useNote.ts

```diff
@@ -87,6 +104,13 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
    */
   const router = useRouter();
 
+  /**
+   * Access rights for currently opened note
+   *
+   * Create new NoteAccessRights instance when new note is created
+   */
+  const accessRights = ref<NoteAccessRights | null>(currentId.value == null ? createAccessRights() : null);
```

Comment:
I think you can simply declare `canEdit` ref with "true" by default. So you won't need `createAccessRights()` method with looks redundant